helm/mds/templates/egress.yaml

```diff
@@ -46,19 +46,3 @@ spec:
   location: MESH_EXTERNAL
 ---
 {{- end }}
-{{- if (eq .Values.fluentbit.enabled true ) }}
-apiVersion: networking.istio.io/v1alpha3
-kind: ServiceEntry
-metadata:
-  name: fluentbit-output
-  namespace: {{ .Release.Namespace }}
-spec:
-  hosts:
-  - {{ .Values.fluentbit.backend.es.host }}
-  ports:
-  - number: {{ .Values.fluentbit.backend.es.port }}
-    name: https
-    protocol: HTTPS
-  location: MESH_EXTERNAL
-  resolution: DNS
-{{- end }}
```

Comment:
We're still going to need the egress in place for an external elasticsearch - it would just be in the logging namespace instead of the mds namespace.

Reply:
ahh right, good call.